CHANGES/6221.doc

```diff
@@ -0,0 +1 @@
+Documented bindings installation on dev environment
```

Comment:
Unfortunately I think we need to only have a project refer to issues in Redmine that are filed under that project's tracker. The reason is at release time when one issue is used in multiple plugins the redmine state shows it as CLOSED - CURRENT RELEASE while the other plugins are still unreleased.

To resolve, please file a one-liner issues describing the lack of docs as a defect against Pulp-ansible in Redmine and then use that issue number for this please. It's kind of inconvenient to do, but I think it's best for our users.